### ISSUE TITLE
Publishable APK

### ISSUE THREAD

**Issue by shaheer-haider** [vite-pwa/nuxt#33]
I have a website and I want most part of it in a mobile App.
Is it possible to create its PWA and publishable APK of the Nuxt 3 project using vite-pwa?
And is it possible to detect if it is on the web or running as App on Android?

**Comment by userquin** [comment]
@shaheer-haider 

> Is it possible to create its PWA and publishable APK of the Nuxt 3 project using vite-pwa?

no, vite-plugin-pwa or any pwa integration include it, check [PWA Builder](https://www.pwabuilder.com/), keep in mind you will need to pay to publish it: https://docs.pwabuilder.com/#/builder/quick-start?id=prerequisites

> And is it possible to detect if it is on the web or running as App on Android?

If you're using `@vite-pwa/nuxt` you can use `$pwa.isInstalled`, it will be true if PWA app is installed, it is based on this https://thomashunter.name/posts/2021-12-11-detecting-if-pwa-twa-is-installed: check the logic here https://github.com/vite-pwa/nuxt/blob/main/templates/pwa.client.ts#L18
